addon/locales/en/translations.js

```diff
@@ -46,31 +46,42 @@ export default {
             buttonSubmit: 'Create account'
         },
         navbar: {
+            osf: 'OSF',
+            openScienceFramework: 'Open Science Framework',
             add: 'Add',
-            addAPreprint: 'Add a {{preprintWords.preprint}}',
+            addAPreprint: 'Add a {{preprintWords.Preprint}}',
             browse: 'Browse',
             cancelSearch: 'Cancel search',
+            dashboard: 'Dashboard',
             donate: 'Donate',
             goHome: 'Go home',
+            meetings: 'Meetings',
             myProjects: 'My Projects',
             myQuickFiles: 'My Quick Files',
+            newProjects: 'New Projects',
+            preprints: 'Preprints',
+            registries: 'Registries',
             reviews: 'My Reviewing',
             search: 'Search',
             searchHelp: 'Search help',
+            altSearchOSF: 'Search OSF',
             searchTheOSF: 'Search the OSF',
             sendSearch: 'Send search query',
             support: 'Support',
+            toggleNavigation: 'Toggle navigation',
             togglePrimary: 'Toggle primary navigation',
             toggleSecondary: 'Toggle secondary navigation',
         },
         authDropdown: {
-            logOut: 'Log out',
+            logOut: 'Log Out',
             myProfile: 'My Profile',
+            support: 'Support',
             osfSupport: 'OSF Support',
             settings: 'Settings',
             signUp: 'Sign Up',
-            signIn: 'Sign in',
-            toggleAuthDropdown: 'Toggle auth dropdown'
+            signIn: 'Sign In',
+            toggleAuthDropdown: 'Toggle auth dropdown',
+            altUserGravatar: 'User gravatar'
```

Comment:
Gravatar was generalized in the osf.io codebase to allow it to possibly change in the future. Not sure if that applies here since I didn't look how this is used yet. So this might be better as "user profile image" or something.
https://github.com/CenterForOpenScience/osf.io/pull/7629

if this is used in other repos don't rename.

Reply:
Looks like osf.io is using `User profile image` as their alt.  I'll go ahead and change that here to make it the same :+1: